source/NVDAObjects/UIA/edge.py

```diff
@@ -422,6 +422,13 @@ def getNormalizedUIATextRangeFromElement(self,UIAElement):
 			charInfo.collapse(True)
 		return range
 
+	def _get_name(self):
+		name=super(EdgeNode,self).name
+		# #8466: elements with aria-role=alert set fires live region changed event but does not expose the alert text as its name, wihch can be found in its descendants.
+		if "role" in self.ariaProperties and self.ariaProperties["role"] == "alert":

```

Comment:
I think it might be good to check for the tree interceptor's existence, just to make sure. How about:

`if self.ariaProperties.get("role") == "alert" and self.treeInterceptor:`